CHANGELOG.md

```diff
@@ -24,6 +24,9 @@
   and which are not.  
   [ZevEisenberg](https://github.com/ZevEisenberg)
   [#3040](https://github.com/realm/SwiftLint/pull/3040)
+* Restructure test helpers.  
```

Comment:
I wasn't sure if this belonged under Enhancements, since it's not user-facing.

Reply:
It's fine to list internal-only changes in the changelog, I just typically don't do that myself.

By the way, we typically list the _issue_ a PR is resolving in a changelog entry, not the PR number itself, but this also isn't a big deal.

Reply:
While we’re on the topic, why the two trailing spaces?

Reply:
LOL, I removed the changeling entry and Danger complained 😆